module/users/helper.js

```diff
@@ -274,7 +274,7 @@ module.exports = class UsersHelper {
 				let programsResult = userRelatedProgramsData
 
 				programDetails.data = programsResult
-				programDetails.count = programsResult.length
+				programDetails.count = programCount.length
```

Comment:
@Prajwal17Tunerlabs tested this. is  programCount is an array?!

Reply:
yes @VISHNUDAS-tunerlabs i have tested it, programCount is an array